typescript/phoenix-sdk/src/utils/market.ts

```diff
@@ -596,11 +596,63 @@ export function getMarketSwapOrderPacket({
     matchLimit,
     clientOrderId,
     useOnlyDepositedFunds,
+    lastValidSlot: null,
+    lastValidUnixTimestampInSeconds: null,
   };
 
   return orderPacket;
 }
 
+/**
+ * Returns a Phoenix swap order packet
+ *
+ * @param marketData The `MarketData` for the swap market
+ * @param side The side of the order
+ * @param inAmount The amount of the input token
+ * @param slippage The slippage tolerance in bps (optional, default 0.5%)
+ * @param selfTradeBehavior The self trade behavior (optional, default Abort)
+ * @param matchLimit The match limit (optional)
+ * @param clientOrderId The client order ID (optional)
+ * @param useOnlyDepositedFunds Whether to use only deposited funds (optional)
+ */
+export function getMarketSwapOrderPacketWithTimeInForce({
+  marketData,
+  side,
+  inAmount,
+  lastValidSlot,
+  lastValidUnixTimestampInSeconds,
```

Comment:
These params should be included in the param docs

Function description should say that this is only for TIF order, and if you don't want a TIF order use the regular getMarketSwapOrderPacket